client/src/state/DeckBuilderModule.ts

```diff
@@ -23,20 +24,27 @@ export const deckBuilderStore = vuexModule(rootStore, 'deckbuilder', {
     ) {
       state.decks = [];
       for (let initializer of init) {
-        const sideboard: CardColumn[] =
-            (<DraftCard[][]>Array(NUM_COLUMNS)).fill([]).map(() => []);
-        for (const card of initializer.pool) {
-          sideboard[Math.min(card.definition.cmc, sideboard.length - 1)]
-              .push(card);
+        const stored = localStorage.getItem(
+            `draft|${initializer.draftName}|${initializer.player.seatPosition}`);
+        if (stored) {
+          state.decks.push(JSON.parse(stored));
```

Comment:
If the user created and stored a deck midway through the draft and then reopened the editor later, after they'd drafted some more cards, we'd need to add those cards into the stored deck, right?